comment/models/comments.py

```diff
@@ -17,6 +17,7 @@ class Comment(models.Model):
     edited = models.DateTimeField(auto_now=True)
 
     objects = CommentManager()
+    urlhash = models.CharField(max_length=objects.len_urlhash, unique=True, editable=False)
```

Comment:
`objects` won't be defined here then  `urlhash = models.CharField(max_length=objects.len_urlhash, unique=True, editable=False)`

Reply:
I don't find any reason for using dynamic value of max_length. So please adjust it or move `len_urlhash ` to the model directly. I would love to keep the manager for its original purpose which is defining the model interface with DB.

Reply:
If the user want they can change the suffix and prefix to the `urlhash` according to their choice. Any changes there will be reflected in the model and generate a new migration file.

It would otherwise create an issue if the values are changed.

Reply:
I am not sure if I understood your idea but I believe it's not related to our issue here since I am just requsting to move a variable to another file. You can for example use `max_length=50` and restict the developer to not use more than this if you think that the length might exceeded or as I said move this valiable to utils and use it in managers and models.

Reply:
Okay, if you see the setting file there are variables `COMMENT_URL_PREFIX` and `COMMENT_URL_SUFFIX` which are used to make the urlhash. The length is calculated dynamically accordingly.

But as you said, we may set the `max length=50` and ask the developer to not use suffix, prefix and id length that when combined exceed this length, somewhere in the documentation. Are you okay with it?

Reply:
Yup.